modules/complete-onpremise-cluster/main.tf

```diff
@@ -0,0 +1,23 @@
+module cluster-provisioner  {
+    source = "../k8s-cluster"
+
+    create          = var.create
+    reset           = var.reset
+    upgrade-cluster = var.upgrade-cluster
+    command         = var.command
+}
+module install-helm-cert-manager  {
```

Comment:
I think you can remove "install-helm-" part, just name it cert-manager.

Reply:
done